src/App.tsx

```diff
@@ -1,27 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import * as goodsAPI from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleClick = async (loadedGoods: Promise<Good[]>) => (
+    setGoods(await loadedGoods));
```

Comment:
```suggestion
  const handleClick = async (loadedGoods: Promise<Good[]>) => (
    setGoods(await loadedGoods)
  );
```

Reply:
but handle what kind of click? This name does not give me any additional information